### PR TITLE
docstrings - update str and dict to string and object

### PR DESCRIPTION
**Pull request by pcriadoperez**
- replace {str} with {string}
- replace {dict} with {object}
- replace {[dict]} with {[object]}

### CODE REVIEW

File: js/pro/coinex.js [hunk 1]
```diff
@@ -391,11 +391,11 @@ module.exports = class coinex extends coinexRest {
...
          * @name coinex#watchTrades
          * @description get the list of most recent trades for a particular symbol
-         * @param {str} symbol unified symbol of the market to fetch trades for
+         * @param {string} symbol unified symbol of the market to fetch trades for
          * @param {int|undefined} since timestamp in ms of the earliest trade to fetch
          * @param {int|undefined} limit the maximum amount of trades to fetch
-         * @param {dict} params extra parameters specific to the coinex api endpoint
-         * @returns {[dict]} a list of [trade structures]{@link https://docs.ccxt.com/en/latest/manual.html?#public-trades}
+         * @param {object} params extra parameters specific to the coinex api endpoint
+         * @returns {object} a list of [trade structures]{@link https://docs.ccxt.com/en/latest/manual.html?#public-trades}
```
Comment (samgermain) [inline js/pro/coinex.js:398]: This one should be in an array

File: js/pro/bitfinex2.js [hunk 2]
```diff
@@ -882,11 +882,11 @@ module.exports = class bitfinex2 extends bitfinex2Rest {
...
          * @name bitfinex2#watchOrders
          * @description watches information on multiple orders made by the user
-         * @param {str} symbol unified market symbol of the market orders were made in
+         * @param {string} symbol unified market symbol of the market orders were made in
          * @param {int|undefined} since the earliest time in ms to fetch orders for
          * @param {int|undefined} limit the maximum number of  orde structures to retrieve
-         * @param {dict} params extra parameters specific to the bitfinex2 api endpoint
-         * @returns {[dict]} a list of [order structures]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure
+         * @param {object} params extra parameters specific to the bitfinex2 api endpoint
+         * @returns {object} a list of [order structures]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure
```
Comment (samgermain) [inline js/pro/bitfinex2.js:889]: Should be inside an array `[object]`

File: js/pro/bitfinex2.js [hunk 3]
```diff
@@ -212,11 +212,11 @@ module.exports = class bitfinex2 extends bitfinex2Rest {
...
          * @name bitfinex2#watchMyTrades
          * @description watches information on multiple trades made by the user
-         * @param {str} symbol unified market symbol of the market orders were made in
+         * @param {string} symbol unified market symbol of the market orders were made in
          * @param {int|undefined} since the earliest time in ms to fetch orders for
          * @param {int|undefined} limit the maximum number of  orde structures to retrieve
-         * @param {dict} params extra parameters specific to the bitfinex2 api endpoint
-         * @returns {[dict]} a list of [order structures]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure
+         * @param {object} params extra parameters specific to the bitfinex2 api endpoint
+         * @returns {object} a list of [order structures]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure
```
Comment (samgermain) [inline js/pro/bitfinex2.js:219]: should be in an array `[object]`

File: js/pro/bitfinex2.js [hunk 4]
```diff
@@ -194,11 +194,11 @@ module.exports = class bitfinex2 extends bitfinex2Rest {
...
          * @name bitfinex2#watchTrades
          * @description get the list of most recent trades for a particular symbol
-         * @param {str} symbol unified symbol of the market to fetch trades for
+         * @param {string} symbol unified symbol of the market to fetch trades for
          * @param {int|undefined} since timestamp in ms of the earliest trade to fetch
          * @param {int|undefined} limit the maximum amount of trades to fetch
-         * @param {dict} params extra parameters specific to the bitfinex2 api endpoint
-         * @returns {[dict]} a list of [trade structures]{@link https://docs.ccxt.com/en/latest/manual.html?#public-trades}
+         * @param {object} params extra parameters specific to the bitfinex2 api endpoint
+         * @returns {object} a list of [trade structures]{@link https://docs.ccxt.com/en/latest/manual.html?#public-trades}
```
Comment (samgermain) [inline js/pro/bitfinex2.js:201]: should be in an array `[object]`

File: js/pro/whitebit.js [hunk 5]
```diff
@@ -480,11 +480,11 @@ module.exports = class whitebit extends whitebitRest {
...
          * @name whitebit#watchOrders
          * @description watches information on multiple orders made by the user
-         * @param {str} symbol unified market symbol of the market orders were made in
+         * @param {string} symbol unified market symbol of the market orders were made in
          * @param {int|undefined} since the earliest time in ms to fetch orders for
          * @param {int|undefined} limit the maximum number of  orde structures to retrieve
-         * @param {dict} params extra parameters specific to the whitebit api endpoint
-         * @returns {[dict]} a list of [order structures]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure
+         * @param {object} params extra parameters specific to the whitebit api endpoint
+         * @returns {object} a list of [order structures]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure
```
Comment (samgermain) [inline js/pro/whitebit.js:487]: should be in an array `[object]`

File: js/pro/whitebit.js [hunk 6]
```diff
@@ -381,8 +381,8 @@ module.exports = class whitebit extends whitebitRest {
          * @param {str|undefined} symbol unified market symbol
          * @param {int|undefined} since the earliest time in ms to fetch trades for
          * @param {int|undefined} limit the maximum number of trades structures to retrieve
-         * @param {dict} params extra parameters specific to the whitebit api endpoint
-         * @returns {[dict]} a list of [trade structures]{@link https://docs.ccxt.com/en/latest/manual.html#trade-structure}
+         * @param {object} params extra parameters specific to the whitebit api endpoint
+         * @returns {object} a list of [trade structures]{@link https://docs.ccxt.com/en/latest/manual.html#trade-structure}
```
Comment (samgermain) [inline js/pro/whitebit.js:385]: should be in an array `[object]`

File: js/pro/whitebit.js [hunk 7]
```diff
@@ -310,11 +310,11 @@ module.exports = class whitebit extends whitebitRest {
...
          * @name whitebit#watchTrades
          * @description get the list of most recent trades for a particular symbol
-         * @param {str} symbol unified symbol of the market to fetch trades for
+         * @param {string} symbol unified symbol of the market to fetch trades for
          * @param {int|undefined} since timestamp in ms of the earliest trade to fetch
          * @param {int|undefined} limit the maximum amount of trades to fetch
-         * @param {dict} params extra parameters specific to the whitebit api endpoint
-         * @returns {[dict]} a list of [trade structures]{@link https://docs.ccxt.com/en/latest/manual.html?#public-trades}
+         * @param {object} params extra parameters specific to the whitebit api endpoint
+         * @returns {object} a list of [trade structures]{@link https://docs.ccxt.com/en/latest/manual.html?#public-trades}
```
Comment (samgermain) [inline js/pro/whitebit.js:317]: should be in an array `[object]`

File: js/pro/exmo.js [hunk 8]
```diff
@@ -329,11 +329,11 @@ module.exports = class exmo extends exmoRest {
...
          * @name exmo#watchTrades
          * @description get the list of trades associated with the user
-         * @param {str} symbol unified symbol of the market to fetch trades for
+         * @param {string} symbol unified symbol of the market to fetch trades for
          * @param {int|undefined} since timestamp in ms of the earliest trade to fetch
          * @param {int|undefined} limit the maximum amount of trades to fetch
-         * @param {dict} params extra parameters specific to the exmo api endpoint
-         * @returns {[dict]} a list of [trade structures]{@link https://docs.ccxt.com/en/latest/manual.html?#public-trades}
+         * @param {object} params extra parameters specific to the exmo api endpoint
+         * @returns {object} a list of [trade structures]{@link https://docs.ccxt.com/en/latest/manual.html?#public-trades}
```
Comment (samgermain) [inline js/pro/exmo.js:336]: should be in an array `[object]`

File: js/pro/exmo.js [hunk 9]
```diff
@@ -264,11 +264,11 @@ module.exports = class exmo extends exmoRest {
...
          * @name exmo#watchTrades
          * @description get the list of most recent trades for a particular symbol
-         * @param {str} symbol unified symbol of the market to fetch trades for
+         * @param {string} symbol unified symbol of the market to fetch trades for
          * @param {int|undefined} since timestamp in ms of the earliest trade to fetch
          * @param {int|undefined} limit the maximum amount of trades to fetch
-         * @param {dict} params extra parameters specific to the exmo api endpoint
-         * @returns {[dict]} a list of [trade structures]{@link https://docs.ccxt.com/en/latest/manual.html?#public-trades}
+         * @param {object} params extra parameters specific to the exmo api endpoint
+         * @returns {object} a list of [trade structures]{@link https://docs.ccxt.com/en/latest/manual.html?#public-trades}
```
Comment (samgermain) [inline js/pro/exmo.js:271]: should be in an array `[object]`

**Review by samgermain — CHANGES_REQUESTED** [review]

File: bybit-test.js [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+    while (true) {
+        let orders = await bybit.watchOrders ('ETH/USD:ETH');
+        console.log (orders)
+        orders = await bybit.watchOrders('ETH/USD:ETH', undefined, undefined, params={"stop":true})
+        console.log (orders)
+
+    }
+}
+
+main()
```
Comment (samgermain) [inline bybit-test.js:24]: you added some extra files

**Comment by pcriadoperez** [comment]
I'll close and reopen

**Pull request closed without merging**